content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -160,6 +160,7 @@ Feature | Default | Alpha | Beta | GA
 `PublicKeysECDSA` | `false` | 1.19 | - | -
 `RootlessControlPlane` | `false` | 1.22 | - | -
 `UnversionedKubeletConfigMap` | `true` | 1.22 | 1.23 | 1.25
+`UpgradeAddonsBeforeControlPlane` | `false` | - | - | -
```

Comment:
Thanks @tengqm 
Sorry for the late reply.

It isn't a new feature, it's more like a bugfix. So this feature gate has been marked deprecated since it was created.
So it's not in Alpha stage, and will never be Beta or GA.
For more context: https://github.com/kubernetes/kubernetes/pull/116570#discussion_r1179352015

Reply:
In that case, we should document this in the second table (gates for GA'ed or deprecated feature).

Reply:
OK, will update soon.